### ISSUE TITLE
File name vcf2cytosure, cancer

### ISSUE THREAD

**Issue by christacosta** [Clinical-Genomics/scout#3604]
Hi, 
Balsamic now creates '.CGH' files which is great. When I download the file it has both Starlims ID and Scout case number in the file name. However, when I open the file in cytosure it only says "TUMOR" [FE file: "tumor.txt"] in the program. It would be great and safer to avoid patient confusion if the file name would show up even in the opened cgh file in cytosure.

**Comment by dnil** [comment]
Indeed! Thanks a bunch for writing about it! I'll transfer this issue to Balsamic!
